system/views/admin/layout/userMenu.cfm

```diff
@@ -15,6 +17,12 @@
 	</a>
 
 	<ul class="user-menu dropdown-menu dropdown-yellow dropdown-caret dropdown-close">
+		<li>
+			<a href="#event.buildAdminLink( linkTo="homepage.make", queryString="data=#data#" )#" >
+				<i class="fa fa-home"></i>
+				#translateResource( "cms:homepage.make.link" )#
+			</a>
+		</li>
```

Comment:
I'd suggest that there should be logic around this so it only shows for admin requests.

So, if you are logged in as an admin user, and are viewing a front-end page (and so seeing the admin meu bar), there should be no option to make the front-end page your admin homepage...

Reply:
Agree. Will hide the option from front-end admin bar.